src/content/RepoPage/RepoPage.js

```diff
@@ -1,5 +1,72 @@
+import { Link, DataTableSkeleton, Pagination } from 'carbon-components-react';
 import React from 'react';
 import RepoTable from './RepoTable';
+import { gql } from 'apollo-boost';
+import { Query } from 'react-apollo';
+import React, { useState } from 'react';
+
+const [totalItems, setTotalItems] = useState(0);
```

Comment:
These 3 constants should be declared inside the `RepoPage` const.

Reply:
Changed Done.